### PR TITLE
Modify RemoveUserDevice and DeleteSkillService

### LINKED ISSUE #1337: Modify RemoveUserDevice so that it can be removed by admin
**Actual Behaviour**
Currently only user can remove the connected device.

**Expected Behaviour**
Admin and superadmin shall be able to remove the device connected by the user.
Required for https://github.com/fossasia/susi.ai/pull/2669

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #1337 

Changes: Modified RemoveUserDevice.
Changed implementation of DeleteSkillService because ListPrivateSkillService does not contain the email id of the chatbot creator, contains only the uuid.

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@akshatnitd @fragm3 Please review

File: src/ai/susi/server/api/aaa/RemoveUserDevices.java [hunk 1]
```diff
@@ -75,13 +75,19 @@ public JSONObject getDefaultPermissions(UserRole baseUserRole) {
     }
 
     @Override
-    public ServiceResponse serviceImpl(Query query, HttpServletResponse response, Authorization authorization, JsonObjectWithDefault permissions) throws APIException {
+    public ServiceResponse serviceImpl(Query query, HttpServletResponse response, Authorization rights, JsonObjectWithDefault permissions) throws APIException {
```
Comment (akshatnitd) [inline src/ai/susi/server/api/aaa/RemoveUserDevices.java]: Do not change naming convention here. Instead, change variable name at the other instance.

File: src/ai/susi/server/api/cms/DeleteSkillService.java [hunk 2]
```diff
@@ -49,7 +49,7 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
         String model_name = call.get("model", "general");
         String access_token = call.get("access_token", null);
         String privateSkill = call.get("private", null);
-        String emailId = call.get("email", null);
+        String key = call.get("key", null);
```
Comment (akshatnitd) [inline src/ai/susi/server/api/cms/DeleteSkillService.java]: key isn't the right param name. Make it `uuid`

**Comment by harsh-jindal** [comment]
@akshatnitd Fixed

**Pull request merged**